src/InputSelect/index.jsx

```diff
@@ -50,6 +50,7 @@ class Select extends React.Component {
         onChange={props.onChange}
         onBlur={props.onBlur}
         ref={props.inputRef}
+        disabled={props.disabled}
```

Comment:
Can you add this to [the `Input Select README.md`](https://github.com/edx/paragon/blob/master/src/InputSelect/README.md)

Reply:
No you don't need to do this